layouts/millennium_r4_sidebar.html

```diff
@@ -15,6 +15,12 @@ <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></
           <li><a href="/millennium/r4/encounters/encounter/">Encounter</a></li>
         </ul>
       </li>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/r4/general-clinical/">General Clinical</a></h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/r4/general-clinical/allergy-intolerance/">Allergy Intolerance</a></li>
```

Comment:
```suggestion
          <li><a href="/millennium/r4/general-clinical/allergy-intolerance/">AllergyIntolerance</a></li>
```

Resource name should be one word here.